.gitigone

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
no need to add gitignor file

Reply:
Yes now now need for **.gitignore** file (it include ignore node_modules folder only now) as your library not depend on any packages, But on the future may be depend any library at lest dev-dependencies when you apply unit test on your **validation.js** file and any one will contribute on library may need to npm install this package to work.

See most popular Library [Angular](https://github.com/angular/angular) and [React](https://github.com/facebook/react) Repositories both has **.gitignore** file.